lanelet2_python/python_api/io.cpp

```diff
@@ -1,41 +1,69 @@
 #include <lanelet2_io/Io.h>
 
-#include <boost/python.hpp>
+#include "lanelet2_python/internal/converter.h"
 
-namespace py = boost::python;
+using namespace boost::python;
 using namespace lanelet;
 
+struct DictToConfigurationConverter {
+  DictToConfigurationConverter() { converter::registry::push_back(&convertible, &construct, type_id<io::Configuration>()); }
+  static void* convertible(PyObject* obj) {
+    if (!PyDict_CheckExact(obj)) {  // NOLINT
+      return nullptr;
+    }
+    return obj;
+  }
+  static void construct(PyObject* obj, converter::rvalue_from_python_stage1_data* data) {
+    dict d(borrowed(obj));
+    list keys = d.keys();
+    list values = d.values();
+    io::Configuration attributes;
+    for (auto i = 0u; i < len(keys); ++i) {
+      std::string key = extract<std::string>(keys[i]);
+      bool value = extract<bool>(values[i]);
```

Comment:
When I do this, I'm getting 
```
TypeError: No registered converter was able to produce a C++ rvalue of type lanelet::Attribute from this Python object of type bool
```
I went with bools for simplicity. How about we leave it as it is for now, and if we ever need to add parameters of different types, we can generalize in the future.

Reply:
Sorry, I changed my mind in the middle of writing. I wanted to suggest:
```c++
    std::string value = extract<std::string>(values[i]);
```
As I said, this will take the detour of convering the value from bool to a string and then back to a bool in `Attribute::asBool`. But at least it keeps working if we continue to add new options to the `Configuration` object.